src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        String randomColor = new ColorSupplier().getRandomColor();
+        int randNumbers = new Random().nextInt(100);

```

Comment:
Random and ColorSupplier must be the class fields

use constants instead of numbers
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)